packages/cdk8s/src/_loadurl.js

```diff
@@ -15,13 +15,13 @@ if (!url) {
   process.exit(1);
 }
 
+const purl = parse(url);
+
 try {
   if (fs.lstatSync(url).isFile()) {
     fs.createReadStream(url).pipe(process.stdout);
   }
 } catch (err) {
-  const purl = parse(url);
```

Comment:
Why did you end up undoing this?

Reply:
Because the function getHttpClient() uses purl so it needs to be scoped appropriately. Could alternatively turn getHttpClient into a pure function and move parse line, I'll add a commit with that just in case.

Reply:
👍